course_catalog/tasks_test.py

```diff
@@ -479,3 +481,13 @@ def test_get_podcast_data(mocker):
     mock_pipelines = mocker.patch("course_catalog.tasks.pipelines")
     get_podcast_data.delay()
     mock_pipelines.podcast_etl.assert_called_once()
+
+
+def test_update_enrollments_for_email(mocker):
+    """Verify that update_enrollments_for_email call update_enrollments_for_user job"""
+    user = UserFactory.create()
+    mock_task = mocker.patch(
+        "course_catalog.etl.enrollments.update_enrollments_for_user"
+    )
+    update_enrollments_for_email.delay(user.email)
+    mock_task.assert_called_once_with(user)
```

Comment:
Maybe also add a test to verify that the task is not called if no user is found with a matching email address

```
    update_enrollments_for_email.delay(fake_email)
    mock_task.assert_not_called()
```